code/game/objects/items/dice.dm

```diff
@@ -222,15 +225,19 @@
 
 	var/fake_result = roll(sides)//Daredevil isn't as good as he used to be
 	var/comment = ""
-	if(sides == 20 && result == 20)
-		comment = "NAT 20!"
-	else if(sides == 20 && result == 1)
-		comment = "Ouch, bad luck."
+	if(sides > 1)  // less comment spam
```

Comment:
this should be a define, and should definitely be higher than 1. a d4 giving you a special message for half of the rolls is stupid